### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: policy/modules/kernel/files.if [hunk 1]
```diff
@@ -3848,14 +3848,31 @@ interface(`files_dontaudit_read_etc_runtime_files',`
...
+##	in /etc that are dynamically
+##	created on boot, such as mtab.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain to not audit.
+##	</summary>
+## </param>
+#
+interface(`files_dontaudit_execuite_etc_runtime_files',`
```
Comment (pebenito) [inline policy/modules/kernel/files.if:3871]: It should be "`exec`" instead of "`execute`".

File: policy/modules/system/systemd.te [hunk 2]
```diff
@@ -581,11 +581,13 @@ init_read_script_files(systemd_generator_t)
 init_getattr_all_unit_files(systemd_generator_t)
 init_getattr_all_script_files(systemd_generator_t)
 
+# vmware_vsock
+kernel_request_load_module(systemd_generator_t)
```
Comment (pebenito) [inline policy/modules/system/systemd.te:585]: Nit: please move to the end of the `kernel_*` lines (after `kernel_dontaudit_search_unlabeled`)

File: policy/modules/system/userdomain.if [hunk 3]
```diff
@@ -2532,6 +2532,7 @@ interface(`userdom_manage_user_home_content_files',`
 	')
 
 	manage_files_pattern($1, user_home_t, user_home_t)
+	allow $1 user_home_t:file map;
```
Comment (pebenito) [inline policy/modules/system/userdomain.if:2535]: There is a `mmap_manage_files_pattern()` now.

**Review by pebenito — CHANGES_REQUESTED** [review]
A few minor nits.

Side note: I'd take a patch to add a codespell check in the lint github action, if you're interested.

File: policy/modules/kernel/files.if (hunk 1 above)
Comment (cgzones) [reply policy/modules/kernel/files.if:3871]: changed

File: policy/modules/system/systemd.te (hunk 2 above)
Comment (cgzones) [reply policy/modules/system/systemd.te]: moved

File: policy/modules/system/userdomain.if (hunk 3 above)
Comment (cgzones) [reply policy/modules/system/userdomain.if]: used macro

**Review by pebenito — APPROVED** [review]

**Pull request merged**